website/src/content/main-concepts/proxy-to-external-api.mdx

```diff
@@ -16,74 +18,213 @@ Merchant Center applications can make [authenticated requests](/main-concepts/au
 
 Since your backend server is hosted on a different domain, you can't send authenticated requests to your server from your Custom Application. Well, you can but you don't know if the request comes from an authenticated user or not.
 
-# Validating authenticated requests
+# Proxying requests to the external API
+
+To be able to validate that the request comes from an authenticated user, the Merchant Center API Gateway provides an endpoint that must be used to connect to your external API in a secured manner.
+
+```
+/proxy/forward-to
+```
+
+Requests to that endpoint should additionally pass the following HTTP headers:
```

Comment:
Should as that it's optional and not required?

Reply:
At the end it does not matter much. If you use `v2`, the `X-Project-Key` header is validated. If you use the complementary tools that we built (which are documented here), the headers are automatically set.